api4/system.go

```diff
@@ -444,3 +449,49 @@ func pushNotificationAck(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	ReturnStatusOK(w)
 }
+
+func setServerBusy(c *Context, w http.ResponseWriter, r *http.Request) {
+	if !c.App.SessionHasPermissionTo(c.App.Session, model.PERMISSION_MANAGE_SYSTEM) {
+		c.SetPermissionError(model.PERMISSION_MANAGE_SYSTEM)
+		return
+	}
+
+	// number of seconds to keep server marked busy
+	secs := r.URL.Query().Get("secs")
+	if secs == "" {
+		secs = "3600" // default 1 hour
```

Comment:
Might be better as constant such as `DEFAULT_SERVER_BUSY_SECONDS`

Reply:
Done via https://github.com/mattermost/mattermost-server/pull/13212/commits/d3706fb9ecb362c9d539d373c0f5b2fa147e2442